admin-dev/themes/default/template/controllers/cart_rules/form.js

```diff
@@ -474,7 +474,7 @@ $('#cart_rule_select_1').jscroll().data('jscrollapi').load_scroll(baseHref+'&typ
 $('#cart_rule_select_2').jscroll().data('jscrollapi').load_scroll(baseHref+'&type=selected&search=');
 
 $('.uncombinable_search_filter').typeWatch({
-	captureLength: -1,
+	captureLength: 0,
```

Comment:
Hi,

captureLength must be -1, if not, content is not reload by ajax if field content is empty

Best regards